Makefile

```diff
@@ -287,6 +288,7 @@ verify-extended: check-generate check format test-cov test-cov-clean test-integr
 # Rules for local environment                                       #
 #####################################################################
 
+kind-% kind2-% gardener-%: export IPFAMILY ?= $(IPFAMILY)
```

Comment:
Nit: Why do you use `?=` while generally `=` is used in the `export` statements? Do you expect `IPFAMILY` to be set?

Reply:
If you try to set it via `=`, you get the error message `Recursive variable 'IPFAMILY' references itself (eventually).  Stop.` from `make`... I guess this happens due to the env variable being the same name as the argument.
Nonetheless, i researched a little bit more about this problem and the better fix is using the `:=` (simple assignment) instead of the `?=` assignment.